integration/presto/src/main/prestosql/org/apache/carbondata/presto/CarbondataColumnConstraint.java

```diff
@@ -1,93 +0,0 @@
-/*
- * Licensed to the Apache Software Foundation (ASF) under one or more
- * contributor license agreements.  See the NOTICE file distributed with
```

Comment:
please check and handle for prestodb also

Reply:
only this I think, `getPartitionFilters` is needed for prestodb as it doesn't have partition pruning.

Reply:
done